syntax = "proto3";
package users;

service Users {
    rpc GetUsers(GetUsersRequest) returns (GetUsersResponse) {}
    rpc GetUserById(GetUserByIdRequest) returns (GetUserByIdResponse) {}
}


message User {
    string id = 1;
    string name = 2;
    string email = 3;
}

message GetUsersRequest {}

message GetUsersResponse{
    repeated User user = 1;
}

message GetUserByIdRequest {
    string id = 1;
}

message GetUserByIdResponse {
    User user = 1;
}